lib/challenge.js

```diff
@@ -308,11 +313,17 @@ let nestedArrays = [
 ];
 
 function prompt13() {
-  // YOUR CODE HERE
```

Comment:
When compiling this, I'm getting an error that says: 

> Uncaught TypeError: Cannot read properties of undefined (reading 'length')

Let's link up and chat about that sometime this afternoon.